scripts/src/java/org/oppia/android/scripts/coverage/RunCoverage.kt

```diff
@@ -136,6 +170,19 @@ private fun findTestFile(repoRoot: String, filePath: String): List<String> {
     .map { it.relativeTo(repoRootFile).path }
 }
 
+private fun getReportOutputPath(
+  repoRoot: String,
+  filePath: String,
+  reportFormat: ReportFormat
+): String {
+  val fileWithoutExtension = filePath.substringBeforeLast(".")
+  val defaultFilename = when (reportFormat) {
+    ReportFormat.HTML -> "coverage.html"
+    ReportFormat.MARKDOWN -> "coverage.md"
+  }
+  return "$repoRoot/coverage_reports/$fileWithoutExtension/$defaultFilename"
```

Comment:
Would it be better to have the name as maybe MathModelCoverage.md compared to /MathModel/coverage.md? I am asking in case I am missing something about how the coverage reports are to be used for the combined report in M2.

Reply:
I think I have answered my own question, I generated reports with both formats and the resulting directory structure looks good.
<img width="354" alt="Screenshot 2024-06-27 at 21 39 14" src="https://github.com/oppia/oppia-android/assets/59600948/63897b54-7778-44f6-99a4-2c08a3ec3b08">

Reply:
Thanks @adhiamboperes, hope no changes need to be made then!

Reply:
Yes, I'm happy with the current solution.